### PR TITLE
fixed resolver test

### PR DESCRIPTION
**Pull request by keiya01**

### CODE REVIEW

File: src/useForm.test.tsx [hunk 1]
```diff
@@ -2141,8 +2146,88 @@ describe('useForm', () => {
...
+
+        fireEvent.input(screen.getByRole('textbox'), {
+          target: { name: 'test', value: '' },
+        });
+
+        expect((await screen.findByRole('alert')).textContent).toBe('');
+        expect(mockResolver).toHaveBeenCalled();
+        expect(methods.formState.isValid).toBeFalsy();
+        expect(renderCount).toBe(2);
+      });
```
Comment (keiya01) [inline src/useForm.test.tsx:2230]: I think these test case should be the same result. But these test case have different `isValid` value.
Is this result intentionally?

Comment (bluebill1049) [reply src/useForm.test.tsx:2230]: depend on the mode at useFrom, isValid is made for `onChange` only

Comment (keiya01) [reply src/useForm.test.tsx:2230]: oops ... thank you🙇‍♂️

Comment (bluebill1049) [reply src/useForm.test.tsx:2230]: no, thank you!

**Comment by keiya01** [comment]
I fixed it. Could you review, please?

File: src/useForm.test.tsx [hunk 2]
```diff
@@ -2122,17 +2124,20 @@ describe('useForm', () => {
               },
             },
           };
-        });
+        };
 
         render(<Component resolver={resolver} />);
 
+        methods.formState.isValid;
```
Comment (bluebill1049) [inline src/useForm.test.tsx:2131]: smart 👁️

**Review by bluebill1049 — DISMISSED** [review]
LGTM!

**Comment by bluebill1049** [comment]
Thanks @keiya01 🙏

**Review by bluebill1049 — APPROVED** [review]

**Pull request merged**